apps/frontend/stores/editor.ts

```diff
@@ -26,27 +25,32 @@ interface CodeState {
   setCode: (code: string) => void
 }
 
-type CodeStore = ReturnType<typeof createCodeStore>
+export const createCodeStore = create<CodeState>((set) => ({
+  code: '',
+  setCode: (code) => {
+    set({ code })
+  }
+}))
```

Comment:
```suggestion
export const createCodeStore = () => {
  return createStore<CodeState>()((set) => ({
    code: '',
    setCode: (code) => {
      set({ code })
    }
  }))
}
```

다른 PR이랑 충돌날 것 같아서 zustand의 `create` 사용하는 것 대신 `createCodeStore` 함수에서 `persist`만 없애야 할 것 같아요!